sqlstore/src/test/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateLibraryDaoTest.java

```diff
@@ -324,14 +324,14 @@ public void testListLibraryTypesByPlatform() throws Exception {
   @Test
   public void testListAllLibraryTypes() throws Exception {
     List<LibraryType> libraryTypes = dao.listAllLibraryTypes();
-    assertEquals(18, libraryTypes.size());
+    assertTrue(libraryTypes.size() > 0);
```

Comment:
It would be ideal to clear and repopulate the LibraryType table in the V9000 test data migration. Then it's easier to test against an exact controlled number of LibraryTypes, rather than making the tests more lenient.

Same with LibrarySelectionType below